packages/exchange/test/utils.ts

```diff
@@ -48,13 +51,7 @@ export const depositToken = async (
 ) => {
     const registryWithUserAsSigner = registry.connect(sender);
 
-    await registryWithUserAsSigner.functions.safeTransferFrom(
-        sender.address,
-        to,
-        id,
-        amount,
-        '0x0'
-    );
+    await registryWithUserAsSigner.safeTransferFrom(sender.address, to, id, amount, '0x00');
```

Comment:
Is there a literal for '0x00' in the ether lib? Seems for some reason we need to use '0x00' instead of '0x0'

Reply:
Ethers no longer accepted `0x0` because it's an odd hex number. Not sure why that change happened.